src/main/java/com/polezhaiev/shop/dto/order/order/OrderMakeRequestDto.java

```diff
@@ -0,0 +1,8 @@
+package com.polezhaiev.shop.dto.order.order;
+
+import lombok.Data;
+
+@Data
+public class OrderMakeRequestDto {
+    private String shippingAddress;

```

Comment:
consider adding validation whenever needed.

Reply:
not fixed